packet.go

```diff
@@ -14,6 +13,12 @@ const (
 	ScramblingControlScrambledWithOddKey  = 3
 )
 
+const (
+	MpegTsPacketSize       = 188
```

Comment:
Is there a gain to export those constants?

Reply:
I would leave `MpegTsPacketSize` exported as demuxer (and muxer in future) has an option to change TS packet size and it could be handy to have a constant with default value.

Other two will change.

Reply:
Perfect